src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Event.scala

```diff
@@ -256,4 +256,18 @@ object Event {
     */
   def parse(line: String): DecodeResult[Event] =
     parser.parse(line)
+
+  /**
+    * Creates empty event which optional fields of it are None
+    * Only necessary fields are given as arguments
+    */
+  def emptyEvent(id: UUID, collectorTstamp: Instant, vCollector: String, vEtl: String): Event =
```

Comment:
It doesn't look like this is being used anywhere.

Also, the name `emptyEvent` is confusing. This is in fact a fully valid event, with all required fields. "Empty" suggests a malformed event instead.

Reply:
It is used, just not inside SDK itself. Do you have a better suggestion for its name?

Reply:
I ended up renaming this to `basic`. So the usage would be `Event.basic(args)`.